commands/branch.py

```diff
@@ -0,0 +1,13 @@
+from commands.base import Command
+from helpers import code
+
+import subprocess
+
+GIT_BRANCH = r"git branch | grep \* | sed -rn 's/\*[ ]*(.*)/\1/p'"
+
+
+class Branch(Command):
+    desc = 'Returns what feature branch PCSocBot is currently running on.'
+
+    def eval(self):
+        return "Running on " + code(subprocess.getoutput(GIT_BRANCH))
```

Comment:
Had an issue on OSX. If you care, I used this code:

```
    def eval(self):
        output = subprocess.getoutput("git branch")
        branch = re.search('\n\* (.*?)\n', output).group(1)
        return "Running on " + code(branch))
```

After importing `re`

Reply:
I agree a python based solution that is independent of sed version is better